ngraph/core/include/openvino/core/layout.hpp

```diff
@@ -70,6 +81,10 @@ class OPENVINO_API Layout {
     bool m_dynamic = false;
     int64_t m_left_size = 0;
     int64_t m_right_size = 0;
+
+    friend std::vector<int64_t> layout::find_permutation(const Layout& src_layout,
+                                                         const PartialShape& src_shape,
```

Comment:
why do we need `src_shape` here?

Reply:
Updated, we'll need only rank in future to identify permutation (without negative values)